concourse/pipelines/create-microbosh.yml

```diff
@@ -120,11 +122,14 @@ jobs:
           - -e
           - -c
           - |
-            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > bosh-secrets.yml
-            ls -l bosh-secrets.yml
+            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > generate/bosh-secrets.yml
+            ls -l generate/bosh-secrets.yml
+        outputs:
+        - name: generate
```

Comment:
Also, is there the potential for confusion if an output is named the same as a task?